packages/wiz-ui-react/src/components/base/popup/stories/fixed.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import { StoryObj } from "@storybook/react";
+import { ComponentProps, ReactNode, useRef, useState } from "react";
+
+import { WizHeader } from "../../header";
+import { WizPopup } from "../components";
+
+import { popupContentStyle } from "./common";
+
+const PopupButton = ({
+  children,
+  ...props
+}: ComponentProps<typeof WizPopup> & { children: ReactNode }) => {
+  const [isOpen, setIsOpen] = useState(props.isOpen);
+  const anchor = useRef<HTMLButtonElement | null>(null);
+  return (
+    <>
+      <button ref={anchor} onClick={() => setIsOpen((v) => !v)}>
+        Toggle
+      </button>
+      <WizPopup
+        {...props}
+        anchorElement={anchor}
+        isOpen={isOpen}
+        onClose={() => setIsOpen(false)}
+      >
+        <div style={popupContentStyle}>
+          <p>{children}</p>
+          <button onClick={() => setIsOpen(false)}>Close</button>
+        </div>
+      </WizPopup>
+    </>
+  );
+};
+
+export const FixedStory: StoryObj<typeof WizPopup> = {
+  render: (args) => {
+    return (
+      <div
+        style={{
+          height: "150vh",
+        }}
+      >
+        <WizHeader
+          onToggle={() => {}}
```

Comment:
lint の warning は解消しておくほうが良いです。

Reply:
[9577890](https://github.com/Wizleap-Inc/wiz-ui/pull/980/commits/9577890d312aa60fb899e0973591caf1cd0cbfc7)
対応いたしました！

Reply:
回避が難しい箇所では `eslint-disable` を使う形で良いと思いますが、ここは

```suggestion
          onToggle={() => undefined}
```

とでもしておけば link warning は回避できるので `eslint-disable` しなくて良いんじゃないかと思いました。

Reply:
[44c2b78](https://github.com/Wizleap-Inc/wiz-ui/pull/980/commits/44c2b7810bcbe9246932a6cceb081b5dd5dc0117)
このようなテクニックがあるとは知りませんでした。ありがとうございます！🙏